views/layouts/form.blade.php

```diff
@@ -149,37 +190,37 @@
 
 @section('initialStore')
     window['{{ config('twill.js_namespace') }}'].STORE.form = {
```

Comment:
I have wanted to do this many times, but now I will. We are going to move these store initializers to a helper class as arrays and just print them out here, what do you think?

Reply:
(can be a separate pr)

Reply:
I agree, but let's make sure they are easily changed. I'm sure a lot of users are overriding the layout to adjust them.